doc/adrs/187_add_visibility_tool_on_diagram_nodes.adoc

```diff
@@ -0,0 +1,87 @@
+= ADR-187 - Add visibility tool on diagram nodes
+
+== Context
+
+The Show/Hide menu of the diagram nodes context menu (a.k.a the palette) is difficult to understand for the end-users.
+Furthermore, it forces users to click 3 times to hide a graphical node:
+- right click on the diagram node to open the palette
+- left click on the Show/Hide section
+- click on Hide menu item
+
+== Decision
+
+A new UI should be available in Sirius Web, allowing to easily manage the visibility of diagram nodes.
+This UI would be an icon and would be placed on the top right corner of the node.
+The specifier will tell which node descriptions will have this UI.
+
+=== Backend
+
+==== View DSL Diagram
+
+On a `diagram::NodeDescription`, it will be possible to create a new `diagram::NodeAction`.
+The `diagram::NodeAction` will have three properties:
```

Comment:
Some identifier will also be needed to let the frontend highjack the rendering of the node actions